nodejs/week5/src/backend/api/reservations.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const knex = require("../database");
+const { returning } = require("../database");
+
+router.get("/", async (req, res) => {
+  try {
+    const reservationslist = await knex("reservations").select("*");
+    res.send(reservationslist);
+  } catch (error) {
+    throw error;
+  }
+});
+
+router.get("/:id", async (req, res) => {
+  try {
+    const { id } = parseInt(req.params);
+    const reservationById = await knex("reservations").where({ id });
+    res.json(reservationById);
+  } catch (error) {
+    throw error;
+  }
+});
+
+//Adds a new reservation
+router.post("/", async (req, res) => {
+  const {
```

Comment:
I think you should create a temp object like this in your `meals.js` for the post. Also, do you want to verify that none of these values are `undefined`?